File: w1/home/multiple5/multiple5.c
#include <stdio.h>

#define STEP 5

int main() {
    int max;
    
    scanf("%d", &max);
    
    max -= max % STEP;
    
    for ( int i = 0; i < max; i += STEP ) {
        printf("%d ", i);
    }
    printf("%d\n", max);
    
    return 0;
}

File: w4/php_dz/multiple5.php
<?php
    $STEP = 5;
    $max = rtrim(fgets(STDIN));
    $out = "";

    $max -= $max % $STEP;

    for ( $i = 0; $i < $max; $i += $STEP ) {
        $out .= $i . " ";
    }

    echo $out . $max . PHP_EOL;
?>